python/custreamz/custreamz/deployment/custreamz_local/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#Installs Anaconda3, cudf0.8, Dask by default
+FROM rapidsai/rapidsai:cuda10.0-base-ubuntu16.04
+
+#Installs python-confluent-kafka, ipywidgets, ipykernel
+RUN source activate rapids && \
+    conda update -n base -c defaults conda && \
+    conda install -y -c anaconda ipywidgets && \
+    conda install -y -c anaconda ipykernel && \
+    conda install -c conda-forge python-confluent-kafka 
+
+#Installs streamz
+RUN source activate rapids && \
```

Comment:
Yes, that's one reason. Another is that the Streamz conda package might be outdated, since we've added a few more lines of cuDF-integration code & tests in the last couple of months.